vendor.conf

```diff
@@ -14,7 +14,7 @@ github.com/cpuguy83/go-md2man v1.0.8
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76 # v1.1.0
 github.com/dgrijalva/jwt-go a2c85815a77d0f951e33ba4db5ae93629a1530af
 github.com/docker/distribution 83389a148052d74ac602f5f1d62f86ff2f3c4aa5
-github.com/docker/docker 50e63adf30d33fc1547527a4097c796cbe4b770f
+github.com/docker/docker 9e351d037bfb61a94de4241ced02cfbb7db8e3b3 https://github.com/vdaviot/engine.git
```

Comment:
I will add a `[WIP]` to this PR title as long as the PR is not merged on moby/moby. I think it's not desirable to merge it as is, depending on a personal fork of moby/moby. Same for `swarmkit`.

Reply:
@silvin-lubecki yes, of course, thanks